### PR TITLE
Create Migration and Seeder

### PR DESCRIPTION
**Pull request by thientrankhh**
@szabogaliakos , Migration and seeder create success. You check it =)))

### CODE REVIEW

File: database/migrations/2020_04_23_093439_create_orders_table.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('orders', function (Blueprint $table) {
+            $table->bigIncrements('order_id');
+            $table->decimal('total_price',10,1);
```
Comment (DungLe1603) [inline database/migrations/2020_04_23_093439_create_orders_table.php:18]: order_id?
( The primary of products table is id)

File: database/migrations/2020_04_23_093439_create_orders_table.php [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+            $table->string('customer_name');
+            $table->string('address');
+            $table->string('phone');
+            $table->string('email');
+            $table->string('order_notes');
+            $table->dateTime('order_date');
+            $table->boolean('order_status')->default(0);
```
Comment (DungLe1603) [inline database/migrations/2020_04_23_093439_create_orders_table.php]: Add comments for order_status

File: database/migrations/2020_04_23_093453_create_order_products_table.php [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    {
+        Schema::create('order_products', function (Blueprint $table) {
+            $table->bigIncrements('order_product_id');
+            $table->unsignedBigInteger('order_id');
+            $table->unsignedBigInteger('product_id');
+            $table->integer('quantity');
+            $table->decimal('price',10,1);
+            $table->foreign('product_id')->references('id')->on('products');
+            $table->foreign('order_id')->references('order_id')->on('orders');
+            $table->timestamps();
```
Comment (DungLe1603) [inline database/migrations/2020_04_23_093453_create_order_products_table.php:24]: on delete/update foreign keys?

File: database/migrations/2020_04_23_093453_create_order_products_table.php [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+     */
+    public function up()
+    {
+        Schema::create('order_products', function (Blueprint $table) {
+            $table->bigIncrements('order_product_id');
+            $table->unsignedBigInteger('order_id');
+            $table->unsignedBigInteger('product_id');
+            $table->integer('quantity');
+            $table->decimal('price',10,1);
+            $table->foreign('product_id')->references('id')->on('products');
```
Comment (DungLe1603) [inline database/migrations/2020_04_23_093453_create_order_products_table.php:22]: Why we need price column? 
(having price column in products table)

File: database/migrations/2020_04_23_093453_create_order_products_table.php [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('order_products', function (Blueprint $table) {
+            $table->bigIncrements('order_product_id');
```
Comment (DungLe1603) [inline database/migrations/2020_04_23_093453_create_order_products_table.php]: The pivot table should be order_table (without _s) and not include order_product_id

**Review by DungLe1603 — COMMENTED** [review]
You should use standard name from Laravel
- The nomal table name is plural nouns
- The pivot table name is combination of two table name (sigle noun)

**Review by szabogaliakos — APPROVED** [review]

**Review by szabogaliakos — APPROVED** [review]

**Pull request merged**